Manager/src/main/java/io/raspberrywallet/manager/database/Database.java

```diff
@@ -3,23 +3,25 @@
 import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
-import java.util.ArrayList;
-import java.util.List;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.Getter;
 
 public class Database {
-	
-	private List<WalletEntity> wallets = new ArrayList<WalletEntity>();
+
+	@Getter
+	@JsonProperty("wallet")
+	public WalletEntity wallet = null;
 	
 	public Database() throws IOException {
 		this(false);
 	}
 	
-	public void addWallets(List<WalletEntity> wallets) {
-		this.wallets.addAll(wallets);
+	public void setWallet(WalletEntity wallets) {
+		this.wallet = wallet;
```

Comment:
Here is why it's not working, you assign `this.wallet` to `wallet` instead of parameter `wallets`.
Please use IntelliJ warnings that tell you exactly where is the problem